src/Pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,106 @@
+#shop{
+    background: #8e9eab;
+    background: -webkit-linear-gradient(to bottom, #eef2f3, #8e9eab);
+    background: linear-gradient(to bottom, #eef2f3, #8e9eab); 
+    height: 30vh;
+}
+
+.category--container{
+    max-width: 80%;
+    padding: var(--padding-large);
+    margin: auto;
+}
+
+.category--card{
+    position: relative;
+    height: 30rem;
+    width: 25rem;
+    overflow: hidden;
+    cursor: pointer;
+    margin: var(--margin-medium);
+}
+
+.category--card img{
+    object-fit: cover;
+}
+
+.category--title{
+    position: absolute;
+    bottom: 1%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.category--card:hover{
+    transform: scale(1.1);
+}
+
+.category--card img{
+    opacity: 0.9;
+} 
+
+.shadow--bottom{
+    -moz-box-shadow: inset 0 -60px 60px -60px #000000;
+    -webkit-box-shadow: inset 0 -60px 60px -60px #000000;
+    box-shadow: inset 0 -60px 60px -60px #000000;
```

Comment:
Here too

Reply:
its a one time usecase, gradient colors hence not added them as variables in root